Xamarin.Essentials/Accelerometer/Accelerometer.shared.cs

```diff
@@ -36,10 +34,11 @@ public static void Start(SensorSpeed sensorSpeed)
 
         public static void Stop()
         {
+            if (!IsSupported)
+                throw new FeatureNotSupportedException();
+
```

Comment:
For the sensors, I decided to throw on both the start and stop if it is not supported. Consistency and the fact that some platforms may try something...